pkg/operator/target_config_reconciler.go

```diff
@@ -90,14 +108,20 @@ func (t TargetConfigReconciler) sync(ctx context.Context, syncCtx factory.SyncCo
 		UID:        jobSetOperator.UID,
 	}
 
-	specAnnotations := map[string]string{
-		"jobsetoperators.operator.openshift.io/cluster": strconv.FormatInt(jobSetOperator.Generation, 10),
-	}
+	specAnnotations := map[string]string{}
 
 	_, _, err = t.manageWebhookService(ctx, ownerReference)
 	if err != nil {
 		return err
 	}
+	_, _, err = t.manageCertManagerIssuer(ctx, ownerReference)
+	if err != nil {
+		return err
```

Comment:
Would be also better to logging the issues in here for better troubleshooting?.

Reply:
hmm, I should remove from lws-operator then.

Reply:
maybe this is more what we want wrt cert-manager detection? https://github.com/openshift/jobset-operator/pull/1/commits/b023651921f122c02beaa87fc94c1f5ffe1f6421